sklearn/feature_selection/_variance_threshold.py

```diff
@@ -2,10 +2,12 @@
 # License: 3-clause BSD
 
 import numpy as np
+from numbers import Real
```

Comment:
Please put this import in a first separated group of imports

Reply:
> Please put this import in a first separated group of imports

@jeremiedbb ,Thank you for the suggestions. I've implemented the changes.